geonode/api/resourcebase_api.py

```diff
@@ -66,6 +67,9 @@
                   FILTER_TYPES)
 from .paginator import CrossSiteXHRPaginator
 
+DEFAULT_EAST = 180
+DEFAULT_WEST = -180
```

Comment:
I think it's not required to use variables for those, but if you want to please use `EAST_MAX` instead of `DEFAULT`